### PR TITLE
Readme v2

### PR DESCRIPTION
**Pull request by ghost**
As suggested in #3 I made an update to the readme so that other contributors can install the bot more easily :)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,14 @@
+node_modules/
+
 config.js
-node_modules/
\ No newline at end of file
+config.example.js
+
+package.json
```
Comment (TynesideGroup) [inline .gitignore]: This file lists the project dependencies - it needs to exist in the repo.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,14 @@
+node_modules/
+
 config.js
-node_modules/
\ No newline at end of file
+config.example.js
```
Comment (TynesideGroup) [inline .gitignore]: `config.example.js` serves as an example to other contributors - it needs to exist in the repo.

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,14 @@
...
+
+package.json
+package-lock.json
```
Comment (TynesideGroup) [inline .gitignore]: `package-lock.json` is an important part of post-dependency installation - it ensures other contributors install the exact same dependencies as you have working on your local machine. It should exist in the repo.

File: .gitignore [hunk 4]
```diff
@@ -1,2 +1,14 @@
...
+
+yarn.lock
```
Comment (TynesideGroup) [inline .gitignore]: `yarn.lock` is an important part of post-dependency installation - it ensures other contributors install the exact same dependencies as you have working on your local machine. It should exist in the repo.

File: .gitignore [hunk 5]
```diff
@@ -1,2 +1,14 @@
...
+
+.vscode/iisexpress.json
```
Comment (TynesideGroup) [inline .gitignore]: Simplify `.vscode/iisexpress.json` to `.vscode/` to exclude everything that may exist under that directory.

File: .gitignore [hunk 6]
```diff
@@ -1,2 +1,14 @@
...
+.vs/VSWorkspaceState.json
```
Comment (TynesideGroup) [inline .gitignore]: Simplify all of the `.vs/*` lines to to `.vs/` to exclude everything that may exist under that directory.

File: README.md [hunk 7]
```diff
@@ -2,6 +2,24 @@
...
+Go into the folder where you placed the bot using a command-line tool and run (freshly opened after installing yarn)
+```bash
+yarn
+```
+
+### Setup the config
+Copy the `config.example.js` and create a `config.js`
+replace `login` with your uplay email adres
+replace `password` with your uplay password
+replace `base_url` with the API endpoint of r6stats, for example `https://api.r6stats.com/api/v1/`
```
Comment (TynesideGroup) [inline README.md]: Don't add the example API URL here - remember that this API is deprecated.

File: package.json [hunk 8]
```diff
@@ -18,8 +18,9 @@
     "dev": "nodemon --exec npm run start"
   },
   "dependencies": {
-    "discord.js": "^11.1.0",
-    "r6stats": "git://github.com/R6Stats/r6stats-node.git#v2"
+    "discord.js": "^11.4.2",
+    "r6stats": "git://github.com/R6Stats/r6stats-node.git#v2",
+    "yarn": "^1.15.2"
```
Comment (TynesideGroup) [inline package.json]: Yarn is not a project dependency. If used, it should be installed globally on the users machine. The project will work fine just using npm - we just prefer to use Yarn.

**Comment by TynesideGroup** [comment]
@Dakpan94 See my review comments - please resolve and recommit. Thanks!

**Comment by ghost** [comment]
I updated the branch :)

**Review by TynesideGroup — APPROVED** [review]
All looking good now - thanks!

**Pull request merged**